pkg/service/helper/name.go

```diff
@@ -0,0 +1,40 @@
+package helper
+
+import (
+	"fmt"
+	"time"
+)
+
+// contains the set of function which helps with generating resource names, display names
+
+type Kind string
```

Comment:
"Kind" is a very generic name for a type, let's try something relatively verbose